config/jobs/image-pushing/k8s-staging-releng.yaml

```diff
@@ -1,7 +1,7 @@
 periodics:
 # TODO(justaugustus): Decrease interval to 1hr once we cut build over to k8s-infra.
 #                     ref: https://github.com/kubernetes/release/issues/911
-- interval: 4h
+- interval: 10000h
```

Comment:
any reason to put this value and not 24h or 48h?

Reply:
Because we basically want them to never run, but I'd like to keep them around to serve as a template for the next time we turn them on.